### ISSUE TITLE
Note interactive alternative/complementary projects in discussion

### ISSUE THREAD

**Issue by cgreene** [greenelab/meta-review#47]
The table will focus on broader competitors to the journal workflow. It would be good to note specific alternatives (e.g., distill.pub, idyll, the Calico folks system).

**Comment by dhimmel** [comment]
Let's update this post with a todo list of names:

- [x] bookdown https://github.com/greenelab/meta-review/issues/30
- [x] idyll
- [x] distill.pub (example at https://github.com/thomas-tanay/post--L2-regularization)
- [ ] `publication_template` https://andrewgyork.github.io/publication_template/
- [x] Pandoc Scholar https://doi.org/10.7717/peerj-cs.112
- [x] gh-publisher https://github.com/ewanmellor/gh-publisher
- [ ] Fidus Writer https://arxiv.org/abs/1601.02927
- [x] arXiv of the future https://doi.org/10.22541/au.149693987.70506124
- [x] paper of the future http://doi.org/10.22541/au.148769949.92783646
- [x] paper of the future http://blogs.nature.com/naturejobs/2017/06/01/techblog-c-titus-brown-predicting-the-paper-of-the-future
- [x] Continuous publishing http://blog.martinfenner.org/2014/03/10/continuous-publishing/
- [x] Stencila https://elifesciences.org/labs/c496b8bb/stencila-an-office-suite-for-reproducible-research
- [x] Living Journal of Computational Molecular Science https://livecomsjournal.github.io/about/
- [ ] Manuscripts https://www.manuscriptsapp.com/
- [ ] Open Bases Paper Builder https://github.com/openbases/builder-pdf
- [x] knitcitations https://github.com/cboettig/knitcitations

**Comment by cgreene** [comment]
@dhimmel ☝️ updated your post - hope that was what you wanted.

**Comment by agitter** [comment]
We can scan the last eLife open source community call projects to see which are relevant: https://elifesciences.org/labs/f66b5b23/open-source-in-publishing-community-call-february-22

**Comment by agitter** [comment]
I edited the list with the other perspectives or tools that are in the Discussion section TODOs.  We may resolve some of these by deciding they aren't relevant enough.

**Comment by slochower** [comment]
In a limited context, an alternative to the standard "journal workflow" is LiveCoMS (just getting off the ground now). I don't think this fits perfectly -- because I think they are agnostic about workflow before it arrives at the journal -- but it highlights the changing landscape of how papers evolve after publishing.

- The Living Journal of Computational Molecular Science (LiveCoMS) provides a peer-reviewed home for manuscripts which share best practices in molecular modeling and simulation. These works are **living documents, regularly updated with community input**, and can include perpetual updated reviews, tutorials, comparisons between software packages, and other documents which aim to improve the studies in the field and require ongoing updates. 
https://livecomsjournal.github.io/about/

**Comment by agitter** [comment]
@slochower it seems relevant to me.  Their [Paper Writing as Code Development](https://livecomsjournal.github.io/about/paper_code/) post describes some of the same philosophies we have here.

**Comment by vsmalladi** [comment]
Another one that I found was [Stackedit](https://stackedit.io/)

**Comment by agitter** [comment]
[HackMD](https://hackmd.io/) is another collaborative Markdown editing site.  Once we have a complete list we can decide which of these are in scope.

**Comment by slochower** [comment]
Using your turn of phrase, @agitter , I searched "collaborative markdown editor" on google and got a ton of results (even in incognito mode). For example,
- https://standardnotes.org/extensions/collaborative-editor
- http://etherpad.org/
- https://github.com/tralves/collaborative-markdown-editor
- https://firepad.io/

I'm not sure it will be possible to have a "complete list" and I'm not sure how much due diligence will be required to assert what is and isn't available on each. What do you think?

**Comment by agitter** [comment]
> I'm not sure it will be possible to have a "complete list"

I agree.  If there are this many, we could make a broad statement about many other collaborative editors being available.  I don't know of anyone using these editors for manuscripts, they seem oriented toward collaborative note taking or other tasks.

**Comment by agitter** [comment]
I added Stencila to the list above

**Comment by slochower** [comment]
Given that we have collectively listed around ~10 alternatives here, I'm wondering if these should be addressed separately and directly... or if we should craft a general paragraph and then add in the relevant citations where they apply. Thoughts?

**Comment by agitter** [comment]
@slochower I suggest that we handle them briefly and group similar ideas together as much as possible.  That is the approach I started in #69.  We can give credit and pointers to related work without reviewing it in full.

We may also decide that we don't need to cite every tool listed above if they are not sufficiently relevant.

**Comment by agitter** [comment]
I don't see anything in the Fidus Writer [overview](https://www.fiduswriter.org/how-it-works/) that is especially relevant to Manubot so I'm leaving it out for now.  I'm also excluding publication_template for now.  It is a template for writing manuscripts in HTML.

**Comment by agitter** [comment]
I added https://github.com/openbases/builder-pdf to the list above but haven't explored it yet.
